From 0e0ac65b232dfcd9831cedb80fa80f0beef7812a Mon Sep 17 00:00:00 2001
From: Paul Barker <paul@betafive.co.uk>
Date: Sat, 9 Mar 2019 22:33:26 +0000
Subject: [PATCH] kernel-yocto.bbclass: Fix find_patches function

Signed-off-by: Paul Barker <paul@betafive.co.uk>
---
 meta/classes/kernel-yocto.bbclass | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/meta/classes/kernel-yocto.bbclass b/meta/classes/kernel-yocto.bbclass
index bd7a957583..24137ce026 100644
--- a/meta/classes/kernel-yocto.bbclass
+++ b/meta/classes/kernel-yocto.bbclass
@@ -15,10 +15,7 @@ def find_patches(d,subdir):
         patchdir = ''
         if "patchdir" in parm:
             patchdir = parm["patchdir"]
-        if patchdir:
-            if subdir == patchdir:
-                patch_list.append(local)
-        else:
+        if subdir == patchdir:
             patch_list.append(local)
 
     return patch_list
-- 
2.17.1

